LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2016 Intellection Software Ltd
```

Comment:
2016 should be 2017.

And I'm not 100% sure on the `Intellection Software Ltd`